build.sh

```diff
@@ -6,4 +6,28 @@ cd build
 cmake ..
 
 # Run make to build the project. To build utilizing multiple cores, append `-j` and the amount of cores to utilize, for example `make -j8`
-make
\ No newline at end of file
+make
+
+# Run mysql migrations
+
+#Check we have sqlite db
+if [ -f res/CDServer.sqlite ]; then
```

Comment:
Is this shell script in the build folder?  I don't see how this would be checking the right directory if you are in the root of the DarkflameServer codebase.

Reply:
This tripped me up as well. The script is in the root dir of the repo, but the script changes its dir to build, so all paths reflect that. 

Had the paths from the root dir, but after some errors, realized the script was written to change its directory